lib/timescale.rb

```diff
@@ -1,16 +1,18 @@
-require "timescale/version"
 require 'active_record'
-require_relative 'timescale/chunk'
-require_relative 'timescale/hypertable'
-require_relative 'timescale/job'
-require_relative 'timescale/job_stats'
-require_relative 'timescale/continuous_aggregates'
-require_relative 'timescale/compression_settings'
-require_relative 'timescale/hypertable_helpers'
-require_relative 'timescale/migration_helpers'
+
+require 'timescale/acts_as_hypertable'
```

Comment:
Is `require_relative` not working? Any reason to move to `require`?

Reply:
Oops! This was a mistake on my end, these should have remained as `require_relative`, I just forgot to change them back before committing. Good catch